android/app/google-services.json

```diff
@@ -0,0 +1,48 @@
+{
+  "project_info": {
```

Comment:
I think this file should be excluded from GIT, someone may steal your account and charge your card :)

Reply:
My bad, removed